stdlib/REPL/src/REPL.jl

```diff
@@ -427,16 +428,30 @@ LineEditREPL(t::TextTerminal, hascolor::Bool, envcolors::Bool=false) =
         false, false, false, envcolors
     )
 
-mutable struct REPLCompletionProvider <: CompletionProvider end
+mutable struct REPLCompletionProvider <: CompletionProvider
+    modifier::Symbol
+end
+REPLCompletionProvider() = REPLCompletionProvider(:none)
 mutable struct ShellCompletionProvider <: CompletionProvider end
 struct LatexCompletions <: CompletionProvider end
 
+setmodifier!(c::REPLCompletionProvider, val::Symbol) = c.modifier = val
+
 beforecursor(buf::IOBuffer) = String(buf.data[1:buf.ptr-1])
 
 function complete_line(c::REPLCompletionProvider, s::PromptState)
     partial = beforecursor(s.input_buffer)
     full = LineEdit.input_string(s)
     ret, range, should_complete = completions(full, lastindex(partial))
+    if c.modifier === :shift
+        c.modifier = :none
+        # Filter out methods where all arguments are `Any`
```

Comment:
Maybe we want to do this by default, and only *not* filter if the shift modifier is present?